code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -81,6 +81,10 @@
 		user << "<span class='warning'>[src] is empty!</span>"
 		return
 	..()
+	if(!iscyborg(user))
+		desc = "[desc] It's been used up."

```

Comment:
The description isn't needed, it's visible on examine how many units in a reagent container there are.